JustSaying/Messaging/MessageProcessingStrategies/IMessageBackoffStrategy.cs

```diff
@@ -0,0 +1,9 @@
+﻿using JustSaying.Models;
+
+namespace JustSaying.Messaging.MessageProcessingStrategies
+{
+    public interface IMessageBackoffStrategy
+    {
+        int GetVisibilityTimeout(Message message, int approximateReceiveCount);

```

Comment:
This interface is near perfect. Minor detail, could it return a `TimeSpan` and we just take `TotalSeconds` later.